Web Views for Volunteer Home Page

mngshm opened the pull request:
The PR contains the Web Views for FOSS United Volunteer Home Page. 

## Web Template 1 
![image](https://github.com/fossunited/fossunited/assets/82205152/f2e97e7c-ab90-4483-ab9d-91ed236a23ca)

## Web Template 2 - Roles and benefits of Volunteers
- This section consists of Roles and benefits of a volunteer volunteering at FOSS United.
- Lemme know if we have to add more roles and benefits. 
- **Note**: The benefits section is a bit different from what was planned in the Prototype Design. I came up with the above implemented layout/design after experimenting. This looks way more cleaner than in the design I made. 

![image](https://github.com/fossunited/fossunited/assets/82205152/d7e201e3-804b-4d62-9479-0f070cbe8ebc)

## Web Template 3 - Custom CTA
- The last section includes redirection links which will redirect ahead to Meetup Volunteer, Event Volunteer and NGO Volunteer form (which are to be added in the future). 
![image](https://github.com/fossunited/fossunited/assets/82205152/7a53fc9e-bb3c-4321-82f9-860f7540f9e1)

## Whole Page View and Responsive Mode screenshot
![image](https://github.com/fossunited/fossunited/assets/82205152/941abe45-301a-492c-88cb-f45500da1b8f)
![image](https://github.com/fossunited/fossunited/assets/82205152/08daf64b-0043-4a33-818a-8c8e169807ea)

mngshm committed:
Adding FFF Forward Font

mngshm committed:
Merge branch 'fossunited:develop' into volunteer-dev

mngshm committed:
Merge branch 'fossunited:develop' into volunteer-dev

mngshm committed:
feat: Adding Volunteer HomePage Tempalte

mngshm committed:
feat: Adding tempalte for Roles and Benefits Section

mngshm committed:
feat: Adding Custom CTA for redirecting to forms

harshtandiya commented on fossunited/volunteering/web_template/custom_cta/custom_cta.html:
Let's rename it to something more specific, Like "FOSS Volunteer - Custom CTA"

harshtandiya commented:
Designs are very inconsistent with what we already have. How about we let it be re-designed and then merge to develop?

harshtandiya commented on fossunited/volunteering/web_template/custom_cta/custom_cta.html:
The buttons are inconsistent in with, doesn't look good.

mngshm commented:
Yes, lets make all the changes needed.

harshtandiya commented on fossunited/volunteering/web_template/roles_and_benefits/roles_and_benefits.html:
The benefits section can use some more padding

harshtandiya commented on fossunited/volunteering/web_template/volunteers_welcome_page/volunteers_welcome_page.html:
Component is redundant. Already have a template of the same kind

harshtandiya replied:
Please, check the PRs when web templates are added. Also checkout templates in the staging

Mangeshrex commented:
![image](https://github.com/fossunited/fossunited/assets/82205152/5ad5f8c0-ecb0-4bd5-b921-b212024e14af)

Please go through one more time and after that I'll start pushing the changes.

mngshm commented:
![image](https://github.com/fossunited/fossunited/assets/82205152/767b90aa-c32a-451b-a7a9-1ff26d52cfb2)

@harshtandiya  Please go through this once, and then I'll start pushing the changes.

harshtandiya commented:
@Mangeshrex 

- The text in the button is supposed to be inter font.

- I hope the hero image of the globe is not hardcoded. We already have it in landing page, and we should not repeat it here.

- We have a section similar to roles for volunteers. We can just use that. However, you can push this template section too. Just make sure you name it differently. 
![image](https://github.com/fossunited/fossunited/assets/73123690/da35bacc-e155-4481-b53d-f6011623352d)

- Same for the Benefits section. However, the design is definitely different, so we can have this as a different section. Name it likewise. The containers look a little claustrophobic, how about giving a little more padding-x?
![image](https://github.com/fossunited/fossunited/assets/73123690/8fe675f4-6038-42eb-abaa-d80af7b7c48d)

- The buttons in the last section are again not of uniform width. Please change that. Maybe remove the border for the parent container? seems a little tacky?

mngshm commented:
- Will fix the text font, I don't where does the issue actually lie, it was made using the template made by you. will go through it again 
- I don't understand what you mean by "hardcoding the globe", were you expecting me to use jinja templates and make fields for the roles ? 
- Yes, the roles and benefits section is named as "roles and benefits". And I've avoided making smaller and smaller templates. So, I've just made one template of the whole section.

mngshm committed:
fix: applying recommended changes in the templates

mngshm commented:
@harshtandiya have pushed the changes.

mngshm committed:
fix: roles and benefit border and directory tree

mngshm commented:
![image](https://github.com/fossunited/fossunited/assets/82205152/468ec995-ee6b-4a03-86c5-c4681d77a478)

mngshm committed:
fix: icon path, role icon padding

mngshm commented:
![image](https://github.com/fossunited/fossunited/assets/82205152/15d27c20-d7cb-4a17-b81d-012ee69eb0b5)

Final web page view with updated changes including: 
- fixing asset path: using fossunited/public for storing icon svg's 
- fixing the padding: fixed the padding above role icons and added it inside div for brevity 
- deleted FFFForward: because it was useless, as we are using Press2P tentatively.
